<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveToFileForm</class>
 <widget class="QWidget" name="SaveToFileForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save file settings</string>
  </property>
  <widget class="QGroupBox" name="timeMeasureGroupBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Time measure:</string>
   </property>
   <widget class="QRadioButton" name="secondsRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>seconds</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="minutesRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>minutes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hoursRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>hours</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="daysRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>days</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="categorial_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>categorial</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save_pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calncel_pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Save values:</string>
   </property>
   <widget class="QCheckBox" name="y_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>y, amplitude</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="time_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>t, time</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="x_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>x, wavelength</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="save_noramization_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>norm. column</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Decimal separator:</string>
   </property>
   <widget class="QRadioButton" name="comma_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>,      123,4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dot_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>.      123.4</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="experiments_listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>141</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
  </widget>
  <widget class="QLabel" name="experiments_const_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save experiments:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="save_normalized_y_checkBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Normalize y values</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="save_column_titles_checkBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Save column titles</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
